src/components/ScrollTrack/__tests__/ScrollTrack.spec.js

```diff
@@ -76,7 +76,8 @@ it('renders ScrollTrack buttons correctly', () => {
   expect(rightButtonStyle.display).toEqual('none')
 
   // show right arrow
-  track.find(ScrollTrack).node.showRightArrow()
+  track.find(ScrollTrack).instance().showRightArrow()
+  track.update()
```

Comment:
due to changes in Enzyme, `.update()` is sometimes needed after triggering methods, to cause a re-render.